qldpc/codes.py

```diff
@@ -1034,13 +1150,13 @@ def get_hyper_product(
         protograph_z = abstract.Protograph(np.block([mat_In1_H2, mat_H1_Im2_T]))
 
         # identify the qubits to conjugate
-        if conjugate:
-            sector_boundary = protograph_a.group.lift_dim * mat_H1_In2.shape[1]
-            qubits_to_conjugate = slice(sector_boundary, None)
-        else:
-            qubits_to_conjugate = None
+        # if conjugate:
+        #    sector_boundary = protograph_a.group.lift_dim * mat_H1_In2.shape[1]
+        #    qubits_to_conjugate = slice(sector_boundary, None)
+        # else:
+        #    qubits_to_conjugate = None
```

Comment:
Codes like the HGPCode have a "natural" way to determine which qubits should be conjugated.  We want the class to store/remember this data somehow.